schema-engine/sql-migration-tests/tests/migrations/sqlite.rs

```diff
@@ -182,13 +182,9 @@ fn unique_constraint_errors_in_migrations(api: TestApi) {
         .send_unwrap_err()
         .to_user_facing();
 
-    let expected_json = expect![[r#"
```

Comment:
Unrelated, this test depended on source code positioning of the schema push file, as it is displayed in a backtrace. I think that's wrong and changed it.